src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -33,8 +33,9 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
 
 MTR_AVAILABLE(ios(16.1), macos(13.0), watchos(9.1), tvos(16.1))
 @interface MTRDevice : NSObject
-- (instancetype)init NS_UNAVAILABLE;
-+ (instancetype)new NS_UNAVAILABLE;
+// TODO:  implement and error? kmo 2 aug 2024 14h16
```

Comment:
Let's revert these back to how they were

Reply:
note to self:  this was a workaround for the "looking at super" issue for that method specifically, but now it's cropping up on other methods as well.  probably related.